pyfstat/mcmc_based_searches.py

```diff
@@ -116,6 +50,11 @@ class MCMCSearch(BaseSearchClass):
         tp=r"tp",
         argp=r"argp",
     )
+    """
+    symbol_dictionary
```

Comment:
This and the others don't need their name repeated:
![Screenshot_20210107_130923](https://user-images.githubusercontent.com/28396587/103891124-90179a00-50e9-11eb-8d37-f0da16fcba1f.png)